source/includes/media.md

```diff
@@ -30,8 +30,13 @@ const bucket = Cosmic.bucket({
   slug: 'bucket-slug',
   write_key: ''
 })
+
+const mediaObj = req.files[0] // Using Multer
+// OR:
+// const mediaObj = { originalname: filename, buffer: filedata, size: filesize } // Not using Multer
+
 bucket.addMedia({
-  media: '<FILE_DATA>',
+  media: mediaObj,
```

Comment:
Can you please make this consistent with the format throughout the docs which is variables -> `snake_case` Functions/Methods -> `camelCase`?  In this case `media_object`.